examples/choco.py

```diff
@@ -0,0 +1,19 @@
+from pyinfra import host
+from pyinfra.modules import choco
+
+SHELL = 'ps'
```

Comment:
This shouldn't be needed here as each operation should set it individually (as needed). I'll comment lower down on that!

Reply:
Can ignore this is we change the default shell!

Reply:
Ok